recipes/rust-activation/forge_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e -x
+
+## TODO: remove the following `unset` lines, once the following issue in `conda-build` is resolved:
+##       <https://github.com/conda/conda-build/issues/2255>
+
+unset REQUESTS_CA_BUNDLE
+unset SSL_CERT_FILE
+
+rustc --help
+rustdoc --help
+cargo --help
+
+echo "#!/usr/bin/env bash"                         > ./cc
+if [[ ${target_platform} =~ linux.* ]]; then
```

Comment:
I think the `if` can be replaced with this?
```suggestion
echo "$CC \"\$@\"" >> ./cc
```

Reply:
I believe `CC` may not be set at this point. It depends on the order of activation scripts being run for that to work!